packages/e2e-tests/src/tests/Delegation.spec.ts

```diff
@@ -0,0 +1,241 @@
+        AuthMethodType.EthWallet,
+        bobsAuthMethodAuthId,
+        3
+      );
+
+    if (!scopes![AuthMethodScope.SignAnything]) {
+      throw new Error('Bob does not have the "SignAnything" scope on his PKP');
+    }
+
+    // As a dApp owner, create a capacity delegation authSig for Bob's PKP wallet
+    const capacityDelegationAuthSig =
+      // eslint-disable-next-line @typescript-eslint/no-non-null-asserted-optional-chain
+      await alice.createCapacityDelegationAuthSig([bob.pkp?.ethAddress!]);
+
+    // As a dApp owner, delegate the capacity credits NFT to Bob
+    const bobPkpSessionSigs = await devEnv.litNodeClient?.getPkpSessionSigs({
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, @typescript-eslint/no-non-null-asserted-optional-chain
+      pkpPublicKey: bob.authMethodOwnedPkp?.publicKey!,
+      authMethods: [bob.authMethod!],
+      resourceAbilityRequests: [
+        {
+          resource: new LitPKPResource('*'),
+          ability: LitAbility.PKPSigning,
+        },
+        {
+          resource: new LitActionResource('*'),
+          ability: LitAbility.LitActionExecution,
+        },
+      ],
+      capabilityAuthSigs: [capacityDelegationAuthSig],
+    });
+
+    const res = await devEnv.litNodeClient?.executeJs({
+      sessionSigs: bobPkpSessionSigs,
+      code: `(async () => {
+                const sigShare = await LitActions.signEcdsa({
+                toSign: dataToSign,
+                publicKey,
+                sigName: "sig",
+                });
+            })();`,
+      jsParams: {
+        dataToSign: alice.loveLetter,
+        publicKey: bob.authMethodOwnedPkp?.publicKey,
+      },
+    });
+
+    devEnv.releasePrivateKeyFromUser(alice);
+    devEnv.releasePrivateKeyFromUser(bob);
+
+    // -- Expected output:
+    // {
+    //   claims: {},
+    //   signatures: {
+    //     sig: {
+    //       r: "00fdf6f2fc3f13410393939bb678c8ec26c0eb46bfc39dbecdcf58540b7f9237",
+    //       s: "480b578c78137150db2420669c47b220001b42a0bb4e92194ce7b76f6fd78ddc",
+    //       recid: 0,
+    //       signature: "0x00fdf6f2fc3f13410393939bb678c8ec26c0eb46bfc39dbecdcf58540b7f9237480b578c78137150db2420669c47b220001b42a0bb4e92194ce7b76f6fd78ddc1b",
+    //       publicKey: "0465BFEE5CCFF60C0AF1D9B9481B680C2E34894A88F68F44CC094BA27501FD062A3C4AC61FA850BFA22D81D41AF72CBF983909501440FE51187F5FB3D1BC55C44E",
+    //       dataSigned: "7D87C5EA75F7378BB701E404C50639161AF3EFF66293E9F375B5F17EB50476F4",
+    //     },
+    //   },
+    //   decryptions: [],
+    //   response: undefined,
+    //   logs: "",
+    // }
+
+    // -- assertions
+    expect(res?.signatures?.sig.r).toBeDefined();
+    expect(res?.signatures?.sig.s).toBeDefined();
+    expect(res?.signatures?.sig.dataSigned).toBeDefined();
+    expect(res?.signatures?.sig.publicKey).toBeDefined();
+
+    // -- signatures.sig.signature must start with 0x
+    expect(res?.signatures.sig.signature.startsWith('0x')).toBeDefined();
```

Comment:
```suggestion
    expect(res?.signatures.sig.signature.startsWith('0x')).toBe(true);
```

`false` is also defined

Reply:
Good catch!